.github/workflows/external.yml

```diff
@@ -0,0 +1,33 @@
+# Temporary workflow configuration, will become a daily.
+
+name: Daily-External
+
+on: pull_request
```

Comment:
can you state your reasoning putting these in a separate CI file?
specifically, do we also want to run this on daily schedule and pushes? or just PRs?

i suppose that in practice, we do have PRs for releases, so that's covered.
and the double trigger we have today for both push and PR causes each PR to be executed multiple times (when pushed into the author's local repo, as part of the PR, and again on unstable when merged).

so maybe the only thing missing is some daily cron job, so that when we see a failure on some PR, we have a certainty that before that PR the test used to pass.

WDYT?

Reply:
It was just easier to push this way. I thought those jobs would just be part of daily, although that might be a bit too late in many cases and we may want it on the PR.

Reply:
i agree that daily CI is too late and we wanna catch it when a PR is made, before merging it.
i'm arguing that we may want it in daily too.
possibly for 2 reasons:
1. if there's some flakiness or timing issues, maybe daily will expose it.
2. when we see a PR that failed that CI job, we may want some reassurance that before these changes they used to pass, so maybe the daily CI provides that reassurance.

if that makes some sense, i suppose we just need to add a cron trigger at the top of that file.